iOS/issueTrackerApp/issueTrackerApp/Presentation/UI/UIView/Issue/TagLabel.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TagLabel.swift
+//  issueTrackerApp
+//
+//  Created by 조중윤 on 2021/06/09.
+//
+
+import UIKit
+
+class TagLabel: UILabel {
+
+    init() {
+        super.init(frame: .zero)
+        self.layer.cornerRadius = 10
+        self.layer.masksToBounds = true
+    }
+
+    required init?(coder: NSCoder) {
```

Comment:
지원하지 않는다면 아래와 같이 사용한다면 컴파일에서 경고나 오류로 표시할 수 있습니다. 참고만 해주세요!

```suggestion
    @available(*, unavailable, message: "TagLabel는 코드로만 생성할 수 있습니다.")
    required init?(coder: NSCoder) {
```